dev/com.ibm.ws.logging.hpel/src/com/ibm/ws/logging/internal/hpel/HpelLogProviderImpl.java

```diff
@@ -46,6 +46,12 @@ public void configure(Map<String, String> config, File logLocation, TextFileOutp
         HpelConfigurator.init(loggingConfig);
     }
 
+    @Override
+    public void update(Map<String, Object> config) {
+        TrConfigurator.update(config);
```

Comment:
I see no updates to TrConfigurator or FFDCConfigurator that would use the updated key.  Not sure how this flows down.

Reply:
TrConfigurator.update(Map<String, Object>) has a line `config.update(newConfig);` which calls `LogProviderConfigImpl.update(Map<String, Object> -> LogProviderConfigImpl.doCommonInit(Map, boolean))` where I made changes.